velox/dwio/parquet/writer/Writer.h

```diff
@@ -108,6 +109,7 @@ struct WriterOptions : public dwio::common::WriterOptions {
   /// Timestamp time zone for Parquet write through Arrow bridge.
   std::optional<std::string> parquetWriteTimestampTimeZone;
   bool writeInt96AsTimestamp = false;
+  std::optional<std::string> parquetDataPageVersion = std::nullopt;
```

Comment:
Actually we can do better than `std::string` here, how about this?
```c++
enum class DataPageVersion {
  kDefault, V1, V2,
} parquetDataPageVersion = DataPageVersion::kDefault;
```

Reply:
Makes sense. I was originally directly using the `arrow::ParquetDataPageVersion` all the way top-down, but after the refactors, it didn't make sense to import that up.

Reply:
@svm1 Can you add an enum here as suggested above?

Reply:
Added, thanks.